_ont/ont-sercomm-fg1000b-11.md

```diff
@@ -60,6 +219,106 @@ curl -i -s -k -X $'POST' -H $'Content-Type: application/x-www-form-urlencoded' \
     $'http://192.168.100.1/ONT/client/data/Router.json'
 ```
 
+Or via command line with:
+```
+/usr/bin/cmld_client set InternetGatewayDevice.WANDevice.1.X_SC_GponInterfaceConfig.X_SC_Password=00000XXXXXXXXXXXXXXX
+/usr/bin/cmld_client save
+```
+
+## Getting/Setting ONU GPON LOID and LOID password
+{% include alert.html content="Not tested but seems used by the misc config at smd init" alert="Warning" icon="svg-warning" color="red" %}
+
+```
+/usr/bin/cmld_client set InternetGatewayDevice.X_SC_MiscCfg.GPON.LoIdPassword=
+
+/usr/bin/cmld_client set InternetGatewayDevice.X_SC_MiscCfg.GPON.LoId=
+```
+
+## Getting/Setting OMCI software version (ME 7)
+{% include alert.html content="`get` works, `set` is not tested but seems used by the misc config at smd init" alert="Warning" icon="svg-warning" color="red" %}
+
+```
+/usr/bin/cmld_client get InternetGatewayDevice.X_SC_MiscCfg.GPON.OmciVersion
+```
+or via umci_ctl get/set tool (not tested if config overwrite umci or the other way around)
+```
+/usr/sbin/umci_ctl mib get 7
+```
+
+## Getting/Setting OMCI hardware version (ME 256)
+Default value: `Glasfaser.DTV1`
+```
+/bin/mount -o remount,rw /dev/mtdblock5 /tmp/var_link_dir/ft
+echo "XXXXXXXXXXXXX" > /tmp/var_link_dir/ft/hw_version
+/bin/mount -o remount,ro /dev/mtdblock5 /tmp/var_link_dir/ft
+reboot
+```
+## Getting/Setting OMCI vendor ID (ME 256)
+Default value: `53434F4D`
+{% include alert.html content="`set` option is available with `Class_id`, `Entity_id`, `Index` and `Value` parameters, not tested." alert="Warning" icon="svg-warning" color="red" %}
+
+```
+/usr/sbin/umci_ctl mib get 256
+```
+
+## Getting/Setting OMCI equipment ID (ME 257)
+{% include alert.html content="`set` option is available with `Class_id`, `Entity_id`, `Index` and `Value` parameters, not tested." alert="Warning" icon="svg-warning" color="red" %}
+
+```
+/usr/sbin/umci_ctl mib get 256
+```
+
+# Advanced settings
+
+## Transferring files to the stick
+Since neither `netcat`/`nc` nor `ftp`/`Sftp`/`Ftps` are available the best option is to use `curl` to download file from a webserver on your network over HTTP only.
```

Comment:
Since we can lauch sshd (we need to explain how, IIRC we still need a bind mount over `/bin/sc_cli` (unless downloading a standalone binary of course), we should be able to transfer files with `scp -O`.

There is also `/usr/bin/tftp` (Busybox).